src/model/cloud-runner/remote-client/index.ts

```
@@ -193,10 +234,43 @@ export class RemoteClient {
     await CloudRunnerSystem.Run(`git lfs install`);
     assert(fs.existsSync(`.git`), 'git folder exists');
     RemoteClientLogger.log(`${CloudRunner.buildParameters.branch}`);
-    if (CloudRunner.buildParameters.gitSha !== undefined) {
-      await CloudRunnerSystem.Run(`git checkout ${CloudRunner.buildParameters.gitSha}`);
+    // Ensure refs exist (tags and PR refs)
```

ESLint

[error] Expected line before comment.

(lines-around-comment)